docs/Development/ContributeGuide.md

```diff
@@ -83,6 +83,21 @@ Precautions:
 * Do not use special Unicode chars, e.g., U+FF1A 
 * Do not use the character of dollar (as we will use Latex to generate pdf files)
 
+### Code Formatting
+
+We use the [Spotless
+plugin](https://github.com/diffplug/spotless/tree/main/plugin-maven) together with [google-java-format](https://github.com/google/google-java-format) to format our Java code. You can configure your IDE to automatically apply formatting on saving with these steps:
+
+1. Download the [google-java-format
```

Comment:
it can be installed in IDEA.. 
Preferences -> plugins -> search google-java-format

Reply:
these operations can not be ignored:
https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides

Reply:
How about `More detailed setup manual`?